userspace/falco/stats_writer.cpp

```diff
@@ -659,7 +661,10 @@ void stats_writer::collector::collect(const std::shared_ptr<sinsp>& inspector,
 			                                  num_evts,
 			                                  now,
 			                                  stats_snapshot_time_delta_sec);
-			get_metrics_output_fields_additional(output_fields, stats_snapshot_time_delta_sec);
+			if(!m_first_run) {
+				get_metrics_output_fields_additional(output_fields, stats_snapshot_time_delta_sec);
+			}
+			m_first_run = false;
```

Comment:
Oh didn't see we use `m_first_run` here too!

Reply:
You prefer any changes?

Reply:
I overlooked the comment above earlier today. Regardless I believe not making the first log special is better, the code seems robust enough, tested everything once more.